### PR TITLE
Feat: Added grid and Slider components

### PR DESCRIPTION
**Pull request by sainath7878**
Added grid and slider components
Grid:
Screenshot 2022-02-21 at 12 22 25 PM
Screenshot 2022-02-21 at 12 22 42 PM

Slider:
Screenshot 2022-02-21 at 12 22 59 PM
Screenshot 2022-02-21 at 12 23 12 PM

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -1105,6 +1122,95 @@ <h1 class="fs-l">Rating</h1>
...
+                            <p class="fs-m text-align-center">Grid Item 2</p>
+                        </div>
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 3</p>
+                        </div>
+                    </div>
+                </script>
+            </section>
+
+            <!-- ------ Slider ------  -->
```
Comment (hsnice16) [inline docs.html:1181]: Nice, you have used semantic HTML and maintained the proper space with comments.

**Review by hsnice16 — APPROVED** [review]
Nice work, the code structure looks good. UI is neat and awesome. Great work.

**Review by Dev-AkshatGupta — APPROVED** [review]
It looks cool to me

**Review by anik31 — APPROVED** [review]
Good work @sainath7878 The UI looks great & the code is also clean. You have given enough spacing & comments which makes it clearly readable.

**Pull request merged**